028-HubAndSpoke/Student/05-PaaS.md

```diff
@@ -0,0 +1,37 @@
+# Challenge 5: PaaS Networking
+
+[< Previous Challenge](./04-AppGW.md) - **[Home](README.md)**
+
+## Introduction
+
+In this challenge you will have a deep look at ways of integrating Azure PaaS services with your Virtual Network.
+
+## Description
+
+You are charged to integrate Azure PaaS services into the design. First, the web servers in the spokes need to access each an Azure SQL Database, but with different requirements:
+
+- The VM in spoke1 needs to access its database AzSQL1 using private IP addresses. DB admins onprem should be able to connect to this database using private IP addresses too
+- Only VMs in spoke2 subnets should be able to access the database for spoke2, AzSQL2
+- None of the databases should be accessible from the public Internet
+
+Additionally, there is an Azure Web Application, that needs connectivity to the spoke1 database AzSQL1. The Azure Web App should also use private IP addresses when accessing the database. The Azure Web App itself should be accessible to onprem users using private addresses.
+
+The following diagram summarizes these requirements:
+
+![paas](media/paas.png)
+
+## Success Criteria
+
+1. You have created 2 Azure SQL databases, AzSQL1 and AzSQL2

```

Comment:
I would bold or use backticks to monospace the DB names so that they stand out.

Reply:
Done